### PR TITLE
fix `extraLabels` description in `values.yaml`

### PR DESCRIPTION
**Pull request by weekface**
### Check List 

Tests 

 - No code

Code changes

 - Has documents change

Side effects

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation

### Does this PR introduce a user-facing change?:
 
 ```release-note
NONE
 ```

### CODE REVIEW

File: charts/tidb-cluster/values.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ rbac:
 # clusterName is the TiDB cluster name, if not specified, the chart release name will be used
 # clusterName: demo
 
-# Add additional TidbCluster labels
+# Add extra labels to TidbCluster object(CRD)
```
Comment (tennix) [inline charts/tidb-cluster/values.yaml]:
```suggestion
# Add extra labels to TidbCluster object
```

**Comment by weekface** [comment]
@tennix PTAL

**Pull request merged**